devito/passes/iet/languages/CXX.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+from sympy.printing.cxx import CXX11CodePrinter
+
+from devito.ir import Call, UsingNamespace, BasePrinter
+from devito.passes.iet.langbase import LangBB
+from devito.symbolics.extended_dtypes import c_complex, c_double_complex
+
+__all__ = ['CXXBB']
+
+
+def std_arith(qualifier=None):
+    if qualifier:
```

Comment:
these first four lines could use some tidying up -- not clear. Also, choice of var names debatable